repo/packages/B/beta-mesosphere-jupyter-service/0/package.json

```diff
@@ -0,0 +1,44 @@
+{
+  "packagingVersion": "4.0",
+  "minDcosReleaseVersion": "1.11",
+  "name": "beta-mesosphere-jupyter-service",
```

Comment:
name should be `jupyterlab`. 
Also, the existing jupyterlab package is in community so I don't see a need to create a `beta-*` package.

Reply:
This is a brand new package that is intended to replace `dcos-jupyterlab-service` eventually. 

It's a completely different configuration schema so we don't want to break any existing users of the older service. 

It's name is also aligned with how we're marketing the service.

Reply:
ok. just a pretty long name for a package that's all.